arch/risc-v/src/common/supervisor/Make.defs

```diff
@@ -21,7 +21,7 @@
 # If the NuttX kernel runs in S-mode
 
 CMN_ASRCS += riscv_dispatch_syscall.S
-CMN_CSRCS += riscv_sbi.c riscv_perform_syscall.c
+CMN_CSRCS += riscv_perform_syscall.c
```

Comment:
Oops, I'll provide the file on monday

Reply:
Sure, let's wait your new patch to bring riscv_sbi.c back